python/ray/serve/deployment_state.py

```diff
@@ -336,7 +336,10 @@ def start(self, deployment_info: DeploymentInfo, version: DeploymentVersion):
 
         self._allocated_obj_ref = self._actor_handle.is_allocated.remote()
         self._ready_obj_ref = self._actor_handle.reconfigure.remote(
-            deployment_info.deployment_config.user_config
+            deployment_info.deployment_config.user_config,
+            # ensure that `reconfigure` will only be called after a response
+            # has been received from `is_allocated`.
```

Comment:
```suggestion
            # Ensure that `reconfigure` will only be called after a response
            # has been received from `is_allocated`.
```
nit: capitalization

please also add *why* we care about the ordering here

Reply:
Added a comment on that, but I find it a bit difficult to concisely explain. Feel free to nitpick it if you can think of something clearer!

Reply:
looks great 👍